package.json

```
@@ -33,6 +33,7 @@
     "preventManualRelease": "node scripts/preventManualRelease.js",
     "publish:prerelease": "lerna publish --canary --dist-tag next --preid pre",
     "publish:prerelease:force": "npm run publish:prerelease -- --force-publish",
+    "github:publish:prerelease": "lerna publish --canary --summary-file --dist-tag next",
```

```suggestion
    "publish:prerelease:githubAction": "lerna publish --canary --summary-file --dist-tag next",
```

if you want all publish script tp be grouped